class-gf-razorpay.php

```diff
@@ -153,16 +152,20 @@ public function get_customer_fields()
 
     public function callback()
     {
-        $entryId = $_COOKIE[self::RAZORPAY_ENTRY_ID];
-
-        $entry = GFAPI::get_entry($entryId);
+        $razorpayOrderId = $_COOKIE[self::RAZORPAY_ORDER_ID];
 
         $key = $this->get_plugin_setting(self::GF_RAZORPAY_KEY);
 
         $secret = $this->get_plugin_setting(self::GF_RAZORPAY_SECRET);
 
         $api = new Api($key, $secret);
 
+        $order = $api->order->fetch($razorpayOrderId);
```

Comment:
@captn3m0 Why make a request to fetch the order? Can the receipt id not be stored as a session / cookie?